draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -534,14 +546,16 @@
          destination without any errors introduced by hardware noise along the
          way. This is verified at various levels through a variety of error
          detection and correction mechanisms. Since we cannot read or copy a
-         quantum state error detection and correction is more involved.</t>
+         quantum state, error detection and correction is more involved.</t>
 
          <t>To describe the quality of a quantum state, a physical quantity
          called fidelity is used <xref target="NielsenChuang" />. Fidelity takes a
          value between 0 and 1 -- higher is better, and less than 0.5 means the
          state is unusable. It measures how close a quantum state is to the
-         state we have tried to create. It expresses the probability that one
-         state will pass a test to identify as the other. Fidelity is an
+         state we have tried to create. It expresses the probability measuring
+         the actual state yields the same result as measuring the desired target 
```

Comment:
Don't you think this may be splitting the hair too finely for a classical network engineer and making the explanation less clear rather than more clear the process?  I thought it was rather obvious from the context that we are doing the same measurement on the desired state and the target state... Anyway, if you feel strongly about it, I can change the language to what you suggested.

Reply:
I think that at the same time it is important to be correct though. I do think that some variation "It expresses the probability that the state will behave exactly the same as our desired state" or "It expresses the probability that the state will **mimic the behaviour of the** desired state" is a good balance between the original phrase (which was paraphrased from wikipedia) and your suggestion which is not correct in all circumstances.

Reply:
I changed the description of fidelity to the first suggestion that you provided.